proto/cosmos/staking/query.proto

```diff
@@ -7,222 +7,261 @@ import "cosmos/staking/staking.proto";
 
+  // dst_validator_addr defines the validator address to redelegate to.
   bytes dst_validator_addr = 3 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 
+  // pagination defines an optional pagination for the request.
   cosmos.query.PageRequest pagination = 4;
 }
 
-// QueryRedelegationsResponse is response type for the Query/Redelegations RPC method
+// QueryRedelegationsResponse is response type for the Query/Redelegations RPC method.
 message QueryRedelegationsResponse {
   repeated RedelegationResponse redelegation_responses = 1 [(gogoproto.nullable) = false];
 
+  // pagination defines the pagination in the response.
   cosmos.query.PageResponse pagination = 2;
 }
 
-// QueryDelegatorValidatorsRequest is request type for the Query/DelegatorValidators RPC method
+// QueryDelegatorValidatorsRequest is request type for the Query/DelegatorValidators RPC method.
 message QueryDelegatorValidatorsRequest {
+  // delegator_addr defines the delegator address to query for.
   bytes delegator_addr = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
 
+  // pagination defines an optional pagination for the request.
   cosmos.query.PageRequest pagination = 2;
 }
 
-// QueryDelegatorValidatorsResponse is response type for the Query/DelegatorValidators RPC method
+// QueryDelegatorValidatorsResponse is response type for the Query/DelegatorValidators RPC method.
 message QueryDelegatorValidatorsResponse {
+  // validators defines the the validators' info of a delegator.
   repeated Validator validators = 1 [(gogoproto.nullable) = false];
 
+  // pagination defines the pagination in the response.
   cosmos.query.PageResponse pagination = 2;
 }
 
-// QueryDelegatorValidatorRequest is request type for the Query/DelegatorValidator RPC method
+// QueryDelegatorValidatorRequest is request type for the Query/DelegatorValidator RPC method.
 message QueryDelegatorValidatorRequest {
+  // delegator_addr defines the delegator address to query for.
   bytes delegator_addr = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
 
+  // validator_addr defines the validator address to query for.
   bytes validator_addr = 2 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.ValAddress"];
 }
 
-// QueryDelegatorValidatorResponse response type for the Query/DelegatorValidator RPC method
+// QueryDelegatorValidatorResponse response type for the Query/DelegatorValidator RPC method.
 message QueryDelegatorValidatorResponse {
+  // validator defines the the validator info.
   Validator validator = 1 [(gogoproto.nullable) = false];
 }
 
-// QueryHistoricalInfoRequest is request type for the Query/HistoricalInfo RPC method
+// QueryHistoricalInfoRequest is request type for the Query/HistoricalInfo RPC method.
 message QueryHistoricalInfoRequest {
+  // height defines at which height to query the historical info.
   int64 height = 1;
```

Comment:
Why aren't we using uint64 here?

Reply:
We should be.

Reply:
Maybe for another day,

https://github.com/cosmos/cosmos-sdk/blob/69bbb8b327c3cfb967d969bcadeb9b0aef144df6/x/staking/keeper/grpc_query_test.go#L526

And digging deeper brings to tendermint proto files too.